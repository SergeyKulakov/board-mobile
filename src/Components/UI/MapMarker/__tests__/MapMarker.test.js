import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { MapMarker as Component } from '../index'

describe('MapMarker', () => {
  const defaultProps = {
    center: '12.142423/31.321244',
    isActive: true,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
