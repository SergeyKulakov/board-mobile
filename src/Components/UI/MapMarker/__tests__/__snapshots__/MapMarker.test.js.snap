// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapMarker render snapshot 1`] = `
<MapCircle
  center={
    Object {
      "latitude": 12.142423,
      "longitude": 31.321244,
    }
  }
  fillColor="rgba(0, 185, 73, 0.3)"
  radius={300}
  strokeColor="rgba(255, 0, 0, 0.8)"
  strokeWidth={1}
/>
`;
